README.md

```diff
@@ -2086,6 +2086,35 @@ params do
 end
 ```
 
+## Parameter Validation and Coercion using `dry-schema`
+
+As an alternative to the `params` DSL described above, you can use a
```

Comment:
I think we want to add a new section "Parameter Validation and Coercion" then add sub-sections for `params` and `contract`. Talk about this in general in the top section, explaining why both exists and why one may be better.

Reply:
That might be one extra nesting too much. Do you think it's okay for `Built-in Validators` to be `h4` and `allow_blank` (or `With Default`, at the end) to be `h5`?

One alternative is to put the new section under `Parameter Validation and Coercion` at `h3` and describe why you might want to use it instead. It might not hurt to see some user feedback first before making it a first-class option, especially given that `dry-schema`'s development is also stalling a little these days.

Reply:
We might be overthinking it. No strong opinions.